Validation/HGCalValidation/test/python/digiClient_cfg.py

```diff
@@ -1,44 +1,75 @@
+###############################################################################
+# Way to use this:
+#   cmsRun digiClient_cfg.py geometry=D92
+#
+#   Options for geometry D88, D92, D93
+#
+###############################################################################
 import FWCore.ParameterSet.Config as cms
-import os
+import os, sys, imp, re
+import FWCore.ParameterSet.VarParsing as VarParsing
 
-from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
-process = cms.Process('CLIENT',Phase2C9)
+####################################################################
+### SETUP OPTIONS
+options = VarParsing.VarParsing('standard')
+options.register('geometry',
+                 "D88",
+                  VarParsing.VarParsing.multiplicity.singleton,
+                  VarParsing.VarParsing.varType.string,
+                  "geometry of operations: D88, D92, D93")
+
+### get and parse the command line arguments
+options.parseArguments()
+
+print(options)
+
+####################################################################
+# Use the options
+
+if (options.geometry == "D88"):
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
+    process = cms.Process('Client',Phase2C11M9)
+    process.load('Configuration.Geometry.GeometryExtended2026D88Reco_cff')
+    fileInput = 'file:DigiValD88.root'
+elif (options.geometry == "D92"):
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
+    process = cms.Process('Client',Phase2C11M9)
+    process.load('Configuration.Geometry.GeometryExtended2026D92Reco_cff')
+    fileInput = 'file:DigiValD92.root'
+else:
+    from Configuration.Eras.Era_Phase2C11M9_cff import Phase2C11M9
+    process = cms.Process('Client',Phase2C11M9)
+    process.load('Configuration.Geometry.GeometryExtended2026D93Reco_cff')
+    fileInput = 'file:DigiValD93.root'
+
+print("Input file: ", fileInput)
```

Comment:
Wouldn't it be possible to use something like ```'{}'.format(options.geometry)``` to remove all the `ifs`?
That would also make the code more flexible for the future, no?
The last `else` is not checking any condition about which geometry has been supplied, right?

Reply:
Please approve this PR. I am working on using an array and implement it later once tested